ts/components/conversation/ConversationHeader.tsx

```diff
@@ -242,7 +242,7 @@ export class ConversationHeader extends React.Component<PropsType, StateType> {
                     storyViewMode: StoryViewModeType.User,
                   });
                 }
-              : undefined
+              : this.getActionForType()
```

Comment:
It looks like this function would be called a duplicate time from `renderHeader()`.
Also `renderAvatar()` is only called from `renderHeader()`.

What if we change `renderAvatar()` to take the `onClick` handler generated earlier?
It can look like this to indicate it's a fallback handler in case the stories handler isn't needed:
```ts
  private renderAvatar(onClickFallback: undefined | (() => void)): ReactNode {
```

Then it will be simpler and we would not recreate the handler function.